theme/static/js/discover.js

```diff
@@ -1348,6 +1385,9 @@ $(document).ready(function () {
                         initMap(json_point_results, "point_map");
                         setMapItemsList(json_point_results, "point_map", null);
                         $("#discover-point-map-loading-spinner").hide();
+                        var pointMapInitRenderEnd = new Date();
+                        var pointMapInitRenderElapsed = pointMapInitRenderEnd - pointMapInitRenderStart;
+                        console.log("point map init elapsed: " + pointMapInitRenderElapsed);
```

Comment:
same as above and for all similar examples in this file - we shouldn't be committing benchmarking code